lib/metasploit_data_models/version.rb

```diff
@@ -11,6 +11,8 @@ module Version
     MINOR = 2
     # The patch version number, scoped to the {MAJOR} and {MINOR} version numbers.
     PATCH = 5
```

Comment:
I believe I followed the instructions in https://github.com/rapid7/metasploit_data_models/blob/master/CONTRIBUTING.md, which mentions `PATCH` but not what to do with it.  Honestly I just follow the instructions to get rspec to pass.  It may be worth updating CONTRIBUTING.md to describe when/how to change these values.